config/manager/kustomization.yaml

```
@@ -13,4 +13,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator
-  newTag: v1.7.0
+  newTag: 1.8.0  ## OSC_VERSION
```

```suggestion
  newTag: v1.8.0  ## OSC_VERSION
```
Please keep the `v` here to be consistent with Operator SDK.

---

Good catch, @gkurz. Thanks.